lapis2/src/main/kotlin/org/genspectrum/lapis/OpenApiDocs.kt

```diff
@@ -33,6 +35,29 @@ fun buildOpenApiSchema(sequenceFilterFields: SequenceFilterFields, databaseConfi
                     .type("object")
                     .description("valid filters for sequence data")
                     .properties(properties + Pair(MIN_PROPORTION_PROPERTY, Schema<String>().type("number"))),
+            ).addSchemas(
+                REQUEST_SCHEMA_WITH_GROUP_BY_FIELDS,
+                Schema<String>()
+                    .type("object")
+                    .description("valid filters for sequence data")
+                    .properties(
+                        properties + Pair(
+                            "fields",
+                            Schema<String>().type("array").items(Schema<String>().type("string")),
+                        ),
+                    ),
+            ).addSchemas(
+                RESPONSE_SCHEMA_AGGREGATED,
+                Schema<String>()
+                    .type("object")
+                    .description("aggregated sequence data")
+                    .required(listOf("count"))
+                    .properties(
+                        properties +
```

Comment:
Should we add a description here to every field that it is present if and only if it was specified in `fields` in the request?